### DISCUSSION TITLE
希望首页可以显示某个UP的直播状态。

### DISCUSSION THREAD

**Discussion by ghost** [ywmoyue/biliuwp-lite#937] [feature]
### 问题描述

在仅本地关注，账号不关注的情况下，首页可以显示某个UP的直播状态。

### 描述解决方案

我有两种想法：
1、在UP的个人主页显示直播状态，并将该UP的个人主页自定义为首页；
2、参照simple live/聚合直播，增加一个“关注主播”页(仅本地关注，账号不关注)，并将该页自定义为首页，点击按钮可进入直播间。
![屏幕截图 2024-12-13 214443](https://github.com/user-attachments/assets/f4a4d829-d1a0-4197-a322-1410cf338561)

### 代码贡献

否

**Comment by ywmoyue** [comment]
@unheard2010 在现有的首页直播页中加一栏本地关注怎么样

**Reply by ywmoyue** [reply]
@unheard2010  https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.7.13.1402 试试

**Reply by ywmoyue** [reply]
在用户空间页面右键点击关注按钮可以设置本地关注

**Reply by ghost** [reply]
> @unheard2010 在现有的首页直播页中加一栏本地关注怎么样

很好，感谢大佬！
直播状态只能通过封面右下角的热度是否为0来判断，不太直观_(:3 」∠ )_
直播页关注栏下方的各种推荐直播间可以选择不显示吗？就是只显示关注的直播。
